tasks/README.md

```diff
@@ -45,6 +45,7 @@
 |     [POST /tasks](#post-tasks)     | Creates new task  |
 | [PATCH /tasks/:id](#patch-tasksid) |   Updates tasks   |
 | [GET /tasks/:username](#get-tasksusername) |  Returns all tasks of the user |
+| [PATCH /tasks/self/:id](#patch-tasksselfid) |  Changes in own task  |
```

Comment:
Why do we need to explicitly set an `:id` on a `/self` route?

Reply:
We need to update one particular task, right?

Reply:
Having `/self` is confusing in the API

Just have it as `/tasks/:taskid`, and if the logged in user has the authorization, we would have a 2xx, otherwise a 4xx

Reply:
We already have a `PATCH /tasks/:taskId` API which allows appOwners to make changes in anyone's tasks

This route is for users to be able to make changes in their own tasks

Any suggestions here?